packages/test/local-server-stress-tests/package.json

```
@@ -0,0 +1,118 @@
+		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+		"format": "npm run format:biome",
+		"format:biome": "biome check . --write",
+		"format:prettier": "prettier --write . --cache --ignore-path ../../../.prettierignore",
+		"lint": "fluid-build . --task lint",
+		"lint:fix": "fluid-build . --task eslint:fix --task format",
+		"test": "npm run test:mocha",
+		"test:coverage": "c8 npm test",
+		"test:mocha": "mocha \"lib/test/**/*.spec.*js\"  --exit",
+		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha"
+	},
+	"c8": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.*ts",
+			"dist/test/**/*.*js",
+			"lib/test/**/*.*js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.*ts",
+			"dist/**/*.*js",
+			"lib/**/*.*js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
+	"dependencies": {
+		"@fluid-experimental/tree": "workspace:~",
+		"@fluid-internal/client-utils": "workspace:~",
+		"@fluid-internal/mocha-test-setup": "workspace:~",
+		"@fluid-private/stochastic-test-utils": "workspace:~",
+		"@fluid-private/test-dds-utils": "workspace:~",
+		"@fluidframework/aqueduct": "workspace:~",
+		"@fluidframework/build-common": "^2.0.3",
+		"@fluidframework/build-tools": "^0.53.0",
+		"@fluidframework/container-definitions": "workspace:~",
+		"@fluidframework/container-loader": "workspace:~",
+		"@fluidframework/container-runtime": "workspace:~",
+		"@fluidframework/container-runtime-definitions": "workspace:~",
+		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/core-utils": "workspace:~",
+		"@fluidframework/datastore": "workspace:~",
+		"@fluidframework/datastore-definitions": "workspace:~",
+		"@fluidframework/driver-definitions": "workspace:~",
+		"@fluidframework/driver-utils": "workspace:~",
+		"@fluidframework/eslint-config-fluid": "^5.7.3",
+		"@fluidframework/id-compressor": "workspace:~",
+		"@fluidframework/local-driver": "workspace:~",
+		"@fluidframework/map": "workspace:~",
+		"@fluidframework/runtime-definitions": "workspace:~",
+		"@fluidframework/runtime-utils": "workspace:~",
+		"@fluidframework/sequence": "workspace:~",
+		"@fluidframework/server-local-server": "^5.0.0",
+		"@fluidframework/telemetry-utils": "workspace:~",
+		"@fluidframework/test-utils": "workspace:~",
+		"@fluidframework/tree": "workspace:~",
+		"uuid": "^9.0.0"
+	},
+	"devDependencies": {
+		"@biomejs/biome": "~1.9.3",
+		"@types/mocha": "^10.0.10",
+		"@types/node": "^18.19.0",
+		"@types/uuid": "^9.0.2",
+		"c8": "^8.0.1",
+		"cross-env": "^7.0.3",
+		"eslint": "~8.55.0",
+		"mocha": "^10.2.0",
+		"mocha-multi-reporters": "^1.5.1",
+		"prettier": "~3.0.3",
+		"rimraf": "^4.4.0",
+		"ts-loader": "^9.5.1",
+		"typescript": "~5.4.5"
+	},
+	"fluidBuild": {
+		"tasks": {
+			"build:test": [
+				"^tsc",
+				"^api-extractor:commonjs"
+			]
```

Haven't kept totally up-to-date on recent build changes, but since you depend on test exports from map/sequence, I believe you need to declare this here to make incremental build reliable:

```suggestion
			"build:test": [
				"...",
				"@fluidframework/sequence#build:test"
				"@fluidframework/map#build:test"
			]
```

(see analogous setting in sequence which makes sequence's tests dependent on merge-tree tests for similar test export purposes)